chia/server/server.py

```diff
@@ -796,16 +802,18 @@ def accept_inbound_connections(self, node_type: NodeType) -> bool:
             return True
         inbound_count = len([conn for conn in self.get_connections(node_type) if not conn.is_outbound])
         if node_type == NodeType.FULL_NODE:
-            return inbound_count < self.config["target_peer_count"] - self.config["target_outbound_peer_count"]
+            return inbound_count < int(self.config["target_peer_count"]) - int(
+                self.config["target_outbound_peer_count"]
+            )
```

Comment:
Hm, this is more a "make the config a schema" thing instead of something related to this PR, isn't it?

Reply:
I think that my request avoids us expanding the range of values that work.  Previously there would have generally been exceptions around the comparisons of integers against strings of strings were used in the config.  This makes strings start working.

Reply:
@altendky Thoughts about just 86f90fee7c59d715ab473fecda734f7945d97da6 instead?

Reply:
Eehhhh....  I dunno.  That's just kinda masking rather than doing a specific hinting thing about the `Any` stuff.  `assert isinstance(self.config["target_peer_count"], int), "target_peer_count must be an integer"` and the like in each `if` strikes me as more appropriate.  But sure, it also feels gross.  Feel free to bug and coerce me into trying to start into the desert/marshmallow config refactor.  I think that's the only real fix here.  And I guess asserting int'ness is kinda reducing the allowable range where floats would also have worked before...  blah.  Can you be ok with that?  `:]`